### ISSUE TITLE
Update trial documentation to talk about -j

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5994]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#5994|
|Type|enhancement|
|Created|2012-09-17 08:49:49Z|
|Branch|https://github.com/twisted/twisted/tree/trial-j-docs-5994|

doc/core/howto/testing.xhtml should have a small paragraph mentioning multiple processes support, and what to pay attention to for your tests to work properly in that environment (mostly no weird globals).

Attachments:

* [ticket-5994-docs.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/418/418b89e27b617a792dfb9bc5e36880d5e6b64093/05f56054e3a262b4b1fd0670837d4ff16ba5cc49.patch) (1378 bytes) - added by wirehead on 2013-03-18 23:57:34Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5994 5994
type__enhancement enhancement
reporter__therve therve
priority__normal normal
milestone__ 
branch__branches_trial_j_docs_5994 branches/trial-j-docs-5994
branch_author__tomprince tomprince
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1347871789000000 1347871789000000
changetime__1363745709000000 1363745709000000
version__None None
owner__wirehead wirehead

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="wirehead's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|wirehead set owner to wirehead<br>wirehead set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="wirehead's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|wirehead removed owner<br>wirehead set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jerub's avatar" src="https://avatars.githubusercontent.com/u/315461?s=50" width="50" height="50">](https://github.com/jerub)<a name="note_3"></a>|@jerub set owner to wirehead|
|-|-|

Approved, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_4"></a>|@tomprince commented|
|-|-|

(In [37705]) Branching to trial-j-docs-5994.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_5"></a>|@tomprince set status to `closed`|
|-|-|

(In [37723]) Merge trial-j-docs-5994: Document trial's -j option.

Author: wirehead
Reviewers: Jerub
Fixes: [#5994](https://github.com/twisted/twisted/issues/5994)
